mobile/mobile-core/android-process-service.ts

```diff
@@ -174,21 +214,31 @@ export class AndroidProcessService implements Mobile.IAndroidProcessService {
 		return this._devicesAdbs[deviceIdentifier];
 	}
 
-	private parseMultilineResult(input: string, regex: RegExp): number {
-		let selectedNumber: number;
+	private getApplicationIdentifierFromPid(adb: Mobile.IDeviceAndroidDebugBridge, pid: string, psData?: string): IFuture<string> {
+		return ((): string => {
+			psData = psData || adb.executeShellCommand(["ps"]).wait();
+			// Process information will look like this (without the columns names):
+			// USER     PID   PPID  VSIZE   RSS   WCHAN    PC         NAME
+			// u0_a63   25512 1334  1519560 96040 ffffffff f76a8f75 S com.telerik.appbuildertabstest
+			return this.getFirstMatchingGroupFromMultilineResult<string>(psData, new RegExp(`.*?\s+${pid}\s+.*\s+(.*?)$`));
```

Comment:
incorrect, consider the following line:
```
u0_a63   125512 1334  1519560 96040 ffffffff f76a8f75 S com.telerik.appbuildertabstest
```
If I use your RegExp and check for 25512, it will match...